src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -176,10 +178,17 @@ export const Welcome = () => {
         <div className="flex justify-around items-center text-xl py-10 px-10 py-4 max-w-screen-xl mx-auto">
           <h4>Do you have questions or need support?</h4>
           <Button asChild>
-            <Link to="/faq">View FAQ</Link>
+            <Link
+              to="/faq"
+              // eslint-disable-next-line no-constant-condition
+              target={"/faq" ? FAQ_TERGET : undefined}
```

Comment:
The logic is sound, but since this link is exclusively going to the FAQ and not rendered iteratively like the NavLinks in the navigation bar, we could just set target to `FAQ_TARGET` 😄